SonarQube.TeamBuild.PreProcessor/Roslyn/EmbeddedAnalyzerInstaller.cs

```diff
@@ -147,9 +154,9 @@ private void FetchResourceFromServer(Plugin plugin, string targetDir)
         /// </summary>
         public static string GetPluginSpecificDir(string baseDir, Plugin plugin)
         {
-            // Format is: [base]\[plugin.version]
+            // Format is: [base]\[plugin_version]
```

Comment:
I changed the separator from _dot_ to _underscore_:  _dot_ is used as a separator in the version, so this change makes it easier to distinguish between the two parts of the value (particularly if you have packages that end with numerals as our tests do).